src/checkpoint/checkpoint.h

```diff
@@ -81,6 +81,95 @@ struct __wt_ckpt_connection {
     uint64_t last_base_write_gen;
 };
 
+/*
+ * Block modifications from an incremental identifier going forward.
+ */
+/*
+ * At the default granularity, this is enough for blocks in a 2G file.
```

Comment:
Was this comment associated with `WT_BLOCK_MODS_LIST_MIN`?

Reply:
Good catch!